pkgs/by-name/ma/major-mono-display/package.nix

```
@@ -0,0 +1,39 @@
+{ lib, stdenvNoCC, fetchFromGitHub }:
+
+stdenvNoCC.mkDerivation {
+  pname = "major-mono-display";
+  version = "2.000";
+
+  src = fetchFromGitHub {
+    owner = "googlefonts";
+    repo = "majormono";
+    rev = "fae0bb9c728fe082097baedaf23604e290ddac16";
+    hash = "sha256-ishGGr8bY6UjEG/Hn5We8hOO5mcDq/41+DMn+dQGGUA=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm444 -t $out/share/fonts/truetype fonts/*.ttf
+
+    runHook postInstall
+  '';
+
+  meta = {
+    description = "A monospaced geometric sans serif all-uppercase typeface";
```

ℹ️ According to section ["Meta attributes" in pkgs/README.md](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes)
`meta.description` should not start with a definite or indefinite article:
```suggestion
    description = "Monospaced geometric sans serif all-uppercase typeface";
```